lisa/tests/scheduler/load_tracking.py

```diff
@@ -743,6 +749,23 @@ def _test_behaviour(self, signal_name, error_margin_pct, allowed_error_pct):
         phase = task.phases[0]
         cpu = phase.cpus[0]
 
+	# Note: This test-case is only valid if executed at capacity == 1024.
```

Comment:
Broken indentation here?

Reply:
Yes, it looks like I can't do indentation in python right. It looks fine in vim but it is caused by using tabs instead of whitespace.

Reply:
Fixed.